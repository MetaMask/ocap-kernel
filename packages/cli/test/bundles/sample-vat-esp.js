/**
 * Start function for generic test vat.
 *
 * @param {unknown} parameters - Initialization parameters from the vat's config object.
 * @returns {unknown} The root object for the new vat.
 */
export function start(parameters) {
  const name = parameters?.name ?? 'anonymous';
  console.log(`empece vat root objecto "${name}"`);
  return {
    name,
    stuff: `se inici√≥ por ${JSON.stringify(parameters)}`,
  };
}
