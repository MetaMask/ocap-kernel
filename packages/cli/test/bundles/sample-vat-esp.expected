{"moduleFormat":"endoZipBase64","endoZipBase64":"UEsDBAoAAAAAAAAAAACzfS0JXQIAAF0CAAAUAAAAY29tcGFydG1lbnQtbWFwLmpzb257CiAgInRhZ3MiOiBbXSwKICAiZW50cnkiOiB7CiAgICAiY29tcGFydG1lbnQiOiAiQG9jYXAvY2xpLXYwLjAuMCIsCiAgICAibW9kdWxlIjogIi4vdGVzdC9idW5kbGVzL3NhbXBsZS12YXQtZXNwLmpzIgogIH0sCiAgImNvbXBhcnRtZW50cyI6IHsKICAgICJAb2NhcC9jbGktdjAuMC4wIjogewogICAgICAibmFtZSI6ICJAb2NhcC9jbGkiLAogICAgICAibGFiZWwiOiAiQG9jYXAvY2xpLXYwLjAuMCIsCiAgICAgICJsb2NhdGlvbiI6ICJAb2NhcC9jbGktdjAuMC4wIiwKICAgICAgIm1vZHVsZXMiOiB7CiAgICAgICAgIi4vdGVzdC9idW5kbGVzL3NhbXBsZS12YXQtZXNwLmpzIjogewogICAgICAgICAgImxvY2F0aW9uIjogInRlc3QvYnVuZGxlcy9zYW1wbGUtdmF0LWVzcC5qcyIsCiAgICAgICAgICAicGFyc2VyIjogInByZS1tanMtanNvbiIsCiAgICAgICAgICAic2hhNTEyIjogIjU4MDc5MGZjMGZjZmNmOTAyNzBkMTliOTIyY2FkYzRiMGIzMWRlYmYzYzU2ZjQ4ODJmYmViZDkxYWVjMDk5MTMyN2Q4ZGY3ZGZlMDFiNWU3OWIzNWE2MzE1ODRjYTFiMDE2ZjRkMTY2MzI4OWU4OTE0OGNlYmQ5ZjMxN2YxNTg1IgogICAgICAgIH0KICAgICAgfQogICAgfQogIH0KfVBLAwQKAAAAAAAAAAAAs+4sO00DAABNAwAALwAAAEBvY2FwL2NsaS12MC4wLjAvdGVzdC9idW5kbGVzL3NhbXBsZS12YXQtZXNwLmpzeyJpbXBvcnRzIjpbXSwiZXhwb3J0cyI6WyJzdGFydCJdLCJyZWV4cG9ydHMiOltdLCJfX3N5bmNNb2R1bGVQcm9ncmFtX18iOiIoeyAgIGltcG9ydHM6ICRozY9faW1wb3J0cywgICBsaXZlVmFyOiAkaM2PX2xpdmUsICAgb25jZVZhcjogJGjNj19vbmNlLCAgIGltcG9ydE1ldGE6ICRozY9fX19fbWV0YSwgfSkgPT4gKGZ1bmN0aW9uICgpIHsgJ3VzZSBzdHJpY3QnOyAgICRozY9faW1wb3J0cyhbXSk7T2JqZWN0LmRlZmluZVByb3BlcnR5KHN0YXJ0LCAnbmFtZScsIHt2YWx1ZTogXCJzdGFydFwifSk7JGjNj19vbmNlLnN0YXJ0KHN0YXJ0KTsgICAvKipcbiAqIFN0YXJ0IGZ1bmN0aW9uIGZvciBnZW5lcmljIHRlc3QgdmF0LlxuICpcbiAqIEBwYXJhbSB7dW5rbm93bn0gcGFyYW1ldGVycyAtIEluaXRpYWxpemF0aW9uIHBhcmFtZXRlcnMgZnJvbSB0aGUgdmF0J3MgY29uZmlnIG9iamVjdC5cbiAqIEByZXR1cm5zIHt1bmtub3dufSBUaGUgcm9vdCBvYmplY3QgZm9yIHRoZSBuZXcgdmF0LlxuICovXG5mdW5jdGlvbiAgICAgICAgc3RhcnQocGFyYW1ldGVycyl7XG5jb25zdCBuYW1lPXBhcmFtZXRlcnM/Lm5hbWU/Pydhbm9ueW1vdXMnO1xuY29uc29sZS5sb2coIGBlbXBlY2UgdmF0IHJvb3Qgb2JqZWN0byBcIiR7bmFtZX1cImApO1xucmV0dXJue1xubmFtZSxcbnN0dWZmOiBgc2UgaW5pY2nDsyBwb3IgJHtKU09OLnN0cmluZ2lmeShwYXJhbWV0ZXJzKX1gfTtcblxuIH1cbn0pKClcbiIsIl9fbGl2ZUV4cG9ydE1hcF9fIjp7fSwiX19yZWV4cG9ydE1hcF9fIjp7fSwiX19maXhlZEV4cG9ydE1hcF9fIjp7InN0YXJ0IjpbInN0YXJ0Il19LCJfX25lZWRzSW1wb3J0TWV0YV9fIjpmYWxzZX1QSwECHgMKAAAAAAAAAAAAs30tCV0CAABdAgAAFAAAAAAAAAAAAAAApIEAAAAAY29tcGFydG1lbnQtbWFwLmpzb25QSwECHgMKAAAAAAAAAAAAs+4sO00DAABNAwAALwAAAAAAAAAAAAAApIGPAgAAQG9jYXAvY2xpLXYwLjAuMC90ZXN0L2J1bmRsZXMvc2FtcGxlLXZhdC1lc3AuanNQSwUGAAAAAAIAAgCfAAAAKQYAAAAA","endoZipBase64Sha512":"7c137760832a5b520d187759baf7318f5e77470dd4b86febfc7e2ad0685a5475d6e8938e1a66dfb6802aeac9649de8ea1f67b2bbe87cce2273438649ebcf9536"}